### PR TITLE
[ECO-731] sanitize address before insertion into the db

### PR DESCRIPTION
**Pull request by CRBl69**

### CODE REVIEW

File: rust/processor/src/processors/econia_processor.rs [hunk 1]
```diff
@@ -782,7 +792,7 @@ impl ProcessorTrait for EconiaTransactionProcessor {
                                 .expect("Failed to parse MarketAccounts");
                             let map_field = data.get("map").expect("No map field");
                             market_account_handles.push(MarketAccountHandle {
-                                user: resource.address.clone(),
+                                user: strip_hex_number(resource.address.clone())?,
                                 handle: opt_value_to_string(map_field.get("handle"))?,
```
Comment (elliottdehn) [inline rust/processor/src/processors/econia_processor.rs:796]: `fn event_data_to_place_market_order_event` has `integrator` which is an address, as does `fn event_data_to_place_limit_order_event` and `fn event_data_to_place_swap_order_event`. Now that I look at it, `event_data_to_place_swap_order_event` also has `signing_account` which is another address.

**Review by elliottdehn — CHANGES_REQUESTED** [review]
There were a few missing but other than that LGTM

File: rust/processor/src/processors/econia_processor.rs (hunk 1 above)
Comment (CRBl69) [reply rust/processor/src/processors/econia_processor.rs:796]: Just updated.

**Review by alnoki — CHANGES_REQUESTED** [review]
Should handle special case of `0x00000000000` which becomes `0x0` instead of `0x`

**Review by alnoki — CHANGES_REQUESTED** [review]
Please merge https://github.com/econia-labs/aptos-indexer-processors/pull/15

**Review by alnoki — APPROVED** [review]

**Review by elliottdehn — APPROVED** [review]

**Pull request merged**